cosmos/dbt/project.py

```diff
@@ -21,6 +19,16 @@ def create_symlinks(project_path: Path, tmp_dir: Path, ignore_dbt_packages: bool
             os.symlink(project_path / child_name, tmp_dir / child_name)
 
 
+def copy_manifest_for_partial_parse(project_path: Path, tmp_dir: Path) -> None:
```

Comment:
I don't want to be too nitpick-y but can we call this something like this instead given we're not copying the manifest?

```suggestion
def copy_msgpack_for_partial_parse(project_path: Path, tmp_dir: Path) -> None:
```

Reply:
Sure. I think I used manifest because while researching it I found that under the hood that's how it's referred to; the partial parse is just a serialized manifest. But I do see how that can be confusing! I will update when I get a moment.